### ISSUE TITLE
Can't auto generate classes for Italy package (it.example.app)

### ISSUE THREAD

**Issue by safesta** [InsertKoinIO/koin-annotations#110]
**Describe the bug**
I'll take the `@Factory` annotation as an example.
For any app that has a package name which starts with "it.", the `@Factory` annotations (and pretty much every annotation) won't work.
The auto generated file `Default.kt` will not find the package name because "it" has the ParametersHolder type.

**To Reproduce**
1. Create a project with "it.example.app" as package name
2. Create a class with the `@Factory` annotation
3. Build

**Expected behavior**
Build succeed

**Fix**
Just declare a different name in the auto generated variable instead of use the default Kotlin `it` name.

After fix example:
```public fun KoinApplication.defaultModule(): KoinApplication = modules(defaultModule)
public val defaultModule : Module = module {
    factory() { parametersHolder ->
        it.example.app.TestClass()
    }
}
```

**Koin project used and used version (please complete the following information):**
`koin version 3.5.3`
`koin-annotations version 1.3.0`

**Comment by arnaudgiuliani** [comment]
ha yes ... `it` package name 🙄 
thanks for your feedback

**Comment by arnaudgiuliani** [comment]
Fixed in [3f1a616](https://github.com/InsertKoinIO/koin-annotations/commit/3f1a616a3a874808830b93f968bd966d195e07c8)
